applications/accounting/ofbiz-component.xml

```diff
@@ -31,6 +31,8 @@ under the License.
     <entity-resource type="data" reader-name="seed" loader="main" location="data/AccountingPortletData.xml"/>
     <entity-resource type="data" reader-name="seed-initial" loader="main" location="data/AccountingScheduledServiceData.xml"/>
 
+    <service-resource type="model" loader="main" location="servicedef/services.xml"/>
```

Comment:
I suggest to use a current service file instead introduce a new one

Reply:
Thanks Nicolas, for the suggestion.

However, there are more of those arithmetic functions to get such setting used in various xml, groovy and java files. I prefer to have those in a single services.xml file, instead having those spread across many files.

Plus, having a generic 'services.xml' in the accounting component brings the component more in line with others (and the templates for creating a new plugin). Such would enhance the developer (and thus the contributor) experience. 

I trust you find my solution acceptable.